src/js/legacy/mega-menu.js

```diff
@@ -108,10 +108,11 @@ class MegaMenu {
   }
 
   toggleSubMenu(event) {
-    event.stopPropagation();
     const submenus = Array.from(this.menu.querySelectorAll('.vetnav-panel--submenu'));
     const triggers = Array.from(this.menu.querySelectorAll('.vetnav-level2'));
 
+    event.stopPropagation();
```

Comment:
Why does moving `stopPropagation` to after the variable declarations fix things?

Reply:
Sorry, overlooked this before - that was just a stylistic change to have variable declarations at the top.